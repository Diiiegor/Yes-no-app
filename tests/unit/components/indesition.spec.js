import Indesition from "../../../src/components/Indesition";
import {shallowMount} from "@vue/test-utils";

describe('Tests for the indesition component',()=>{
    let wrapper;
    beforeEach(()=>{
        wrapper = shallowMount(Indesition)
    })

    test('Must be match with the snapshot',()=>{
        expect(wrapper.html()).toMatchSnapshot()
    })

})