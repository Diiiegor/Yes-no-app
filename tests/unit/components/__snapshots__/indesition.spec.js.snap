// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for the indesition component Must be match with the snapshot 1`] = `
<!--v-if-->
<div class="bg-dark"></div>
<div class="indesition-container"><input type="text" placeholder="Write a question">
  <p>Please remember to finish with a question mark (?)</p>
  <!--v-if-->
</div>
`;
